test/runtime/ota/mock.js

```diff
@@ -0,0 +1,10 @@
+module.exports.mockRuntime = mockRuntime
+function mockRuntime () {
+  return {
+    setMicMute: () => {},
+    setPickup: () => {},
+    openUrl: () => {},
+    startMonologue: () => {},
+    component: {}
+  }
+}
```

Comment:
This file should be moved to test/helper.

Reply:
It's designed to be a domained mock helper. So it is not necessary to put in public helper scope.

Reply:
But the runtime directory only contains the test files.

Reply:
`test/runtime` doesn't only contains test files. All test files are suffixed by `.test.js`, while domained helper files are normal `.js` files.

Reply:
Okay, tha looks good if the file is in sub-directory.